lib/views/help/search_engines

```diff
@@ -0,0 +1,84 @@
+<% @title = "Search engines" %>
+<%= render :partial => 'sidebar' %>
+<div id="left_column_flip" class="left_column_flip">
+  <h1 id="title">
+    <%= @title %>
+    <a href="#title">#</a>
+  </h1>
+  <p>
```

Comment:
```suggestion
  <p>
    We are often asked about search engines, and how they work. This page is designed to provide some information about 
    how search engines "find" information that is held on our website, and what you can do if you are concerned about 
    their listings.
  </p>
  <p>
    Please keep in mind that we do not control search engines, so we cannot guarantee that they will remove 
    information; but we've provided some information below, as a courtesy. If you need help, the 
    <a href="#unhappy">Information Commissioner's Office (ICO)</a> has a free helpline, and can provide advice.
  </p>
  <p>
    Finally, if you are concerned about information that is held on our website, please see our 
    <%= link_to 'Privacy Notice', help_privacy_path(anchor: 'your_rights') %> for more information.
  </p>
  <hr>
  <h2 id="why">
    Why do they have my data?
    <a href="#why">#</a>
  </h2>
  <p>
```

Adds some boilerplate at the top, to summarise why the page exists, and to provide a clear link back to the privacy notice[^1]

[^1]: we don't know how they found the page, it might be, ironically enough, via a search engine itself.